pkg/controller/admissionchecks/multikueue/clusterprofile.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package multikueue
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	inventoryv1alpha1 "sigs.k8s.io/cluster-inventory-api/apis/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta2"
+)
+
+const (
+	// ManagedByLabel is a label added to MultiKueueCluster objects to indicate
+	// that they are managed by this controller.
+	ManagedByLabel = "kueue.x-k8s.io/managed-by"
```

Comment:
There's already a "managed by kueue" label that you could use here as well - [code](https://github.com/kshalot/kueue/blob/46219d5d4f3201310d65492153af054c309df359/pkg/constants/constants.go#L41-L43).